tests/acceptance-bdd/test_literature_suggestion_form.py

```diff
@@ -0,0 +1,392 @@
+def test_verify_article_record_in_the_detail_page():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Accept article and verify the confirmation message')
+def test_verify_accepted_article():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Use the autocomplete for the supervisor institution')
+def test_autocomplete_supervisor_institution():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Use the autocomplete for the journal title')
+def test_autocomplete_journal_title():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Use the autocomplete for the conference title')
+def test_autocomplete_conference_title():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Use the autocomplete for the affilitaion')
+def test_autocomplete_affilation():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Import an article using the arXiv id')
+def test_import_from_arxiv():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Import an article using the DOI id')
+def test_import_from_doi():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Writing the wrong arXiv id')
+def test_wrong_arxiv():
+    pass
+
+
+def test_correct_arxiv():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Writing the wrong DOI id')
+def test_wrong_doi():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Writing the correct DOI id')
+def test_correct_doi():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Writing a wrong url')
+def test_wrong_url():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Writing a wrong thesis date')
+def test_wrong_thesis_date():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Writing a correct thesis date')
+def test_correct_thesis_date():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Writing a correct url')
+def test_correct_url():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Submit thesis and verify record detail in the list page')
+def test_verify_thesis_record_in_the_list_page():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Submit thesis and verify record detail in the detail page')
+def test_verify_thesis_record_in_the_detail_page():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Accept thesis and verify the confirmation message')
+def test_verify_accepted_thesis():
+    pass
+
+
+@given('I am logged in')
+def author_user(login):
+    pass
+
+
+@when('I go to the literature suggestion page')
+def go_to_create_literature():
+    Arsenic().get(os.environ['SERVER_NAME'] + '/literature/new')
+
+
+@when('I go to the holding panel list page')
+def go_to_holding_pen_literature_list():
+    Arsenic().get(os.environ['SERVER_NAME'] + '/holdingpen/list/?page=1&size=10&status=HALTED&workflow_name=HEP')
+
+
+@when(parsers.cfparse('I click on the button with id {button_id}'))
+def click_generic_button(button_id):
+    WebDriverWait(Arsenic(), 2).until(
```

Comment:
Above you have a timeout of 10 and these ones here have 2. If I am correct this wait is implicit right? So it's the maximum waiting time before error.

You seem to be repeating a lot the code:

```
WebDriverWait(Arsenic(), 2).until(
EC.visibility_of_element_located((By.ID, button_id))
).click()
```

Would be good to have a helper with a default value for the wait and where you just pass the part inside the `until`